pkg/manager-nvme/manager.go

```diff
@@ -483,26 +468,6 @@ func (s *Storage) formatVolume(volumeID string) error {
 	return ec.NewErrNotFound()
 }
```

Comment:
formatVolume must attach the namespace it is formatting before issuing the format as in SetFeature() line 522